cmake/modules/DownloadAsset2.cmake

```diff
@@ -0,0 +1,9 @@
+function(download_asset2 json install_path)
+    add_custom_target(json-target
+        COMMAND python3 ${CMAKE_SOURCE_DIR}/tools/download_asset2.py
+            --json ${json}
+            github
+            --install_dir ${install_path}
+        COMMENT "Download binary assets listed in json file"
+        )
```

Comment:
I assume that "2" in filename means that it will be merged in the nearest future?

Reply:
https://appnroll.atlassian.net/browse/EGD-7947 this one I think ;)

Reply:
tx docs updated :)